java/opendcs/src/main/java/opendcs/dai/EnumDAI.java

```diff
@@ -7,26 +7,40 @@
 
 import decodes.db.DbEnum;
 import decodes.db.EnumList;
+import decodes.decoder.Season;
+import decodes.sql.DbKey;
 import decodes.tsdb.DbIoException;
+import org.opendcs.database.api.OpenDcsDao;
 
 /**
  * Data Access Interface for database-resident enumerations
  * @author mmaloney Mike Maloney
  */
-public interface EnumDAI
-	extends AutoCloseable
+public interface EnumDAI extends AutoCloseable, OpenDcsDao
 {
-	public DbEnum getEnum(String enumName)
+	DbEnum getEnum(String enumName)
 		throws DbIoException;
 
-	public void readEnumList(EnumList top)
+	void readEnumList(EnumList top)
 		throws DbIoException;
 	
-	public void writeEnumList(EnumList enumList)
+	void writeEnumList(EnumList enumList)
 		throws DbIoException;
 	
-	public void writeEnum(DbEnum dbenum)
+	void writeEnum(DbEnum dbenum)
+		throws DbIoException;
+
+	void deleteEnumList(DbKey refListId)
+		throws DbIoException;
+
+	Season getSeason(String abbr)
```

Comment:
DAO layer should not know about Season, instead the translation to/from Season object should happen in the REST API layer